### ISSUE TITLE
test GoGetta /team - check which info has been written to the db

### ISSUE THREAD

**Issue by adambard1** [streetlives/yourpeer.nyc-nextjs#121]
I filled every data field for the Clothing Pantry Service info form of https://test.gogetta.nyc/find/location/bfbe0ef1-0332-456b-9c9b-4552799d60b1
This is what shows on /find having done that
<img width="595" alt="Screenshot 2024-04-04 at 16 52 44" src="https://github.com/streetlives/streetlives_v2/assets/13530245/bd0491b5-6336-46d4-9350-1eda78053da2">

In the SSTT after saving the data, these fields _don't_ retain it at all
- https://test.gogetta.nyc/team/location/bfbe0ef1-0332-456b-9c9b-4552799d60b1/services/db90dadb-0b34-43ab-bdaa-88800820cbb1/who-does-it-serve
- /gender-and-age

Though data is visible locally if you click into all the other fields, this is the Service info form view - suggesting several other fields haven't retained info (even though they are visible when clicking in)

<img width="509" alt="Screenshot 2024-04-04 at 16 46 15" src="https://github.com/streetlives/streetlives_v2/assets/13530245/9fb85da8-3a20-4747-bdb7-4559675f9d66">

<img width="512" alt="Screenshot 2024-04-04 at 16 46 36" src="https://github.com/streetlives/streetlives_v2/assets/13530245/67a4e8ed-ab5e-4883-8dae-b61a3cd40efb">

**Comment by doobneek1** [comment]
Yes it retains data just does not show it was filled out unless you click, only age group data is not retained @adambard1
